### PR TITLE
Feature sitemap inclusion

### PR DESCRIPTION
**Pull request by adonisjoe**
Screenshot 2023-08-02 at 10 24 10 AM

### CODE REVIEW

File: sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<!-- created with Free Online Sitemap Generator www.xml-sitemaps.com -->
+
+
+<url>
+  <loc>https://test.tisamed-oradea.ro/</loc>
+  <lastmod>2023-08-01T21:35:23+00:00</lastmod>
+  <priority>1.00</priority>
+</url>
+<url>
+  <loc>https://test.tisamed-oradea.ro/index.html</loc>
```
Comment (georgebrata) [inline sitemap.xml:16]: Please use the live production domain (without test).

Also please use the version without index.html at the end, like this: `https://tisamed-oradea.ro/`

File: sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+      xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<!-- created with Free Online Sitemap Generator www.xml-sitemaps.com -->
```
Comment (georgebrata) [inline sitemap.xml:7]: Please remove this comment

File: sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  <lastmod>2023-08-01T21:35:23+00:00</lastmod>
+  <priority>1.00</priority>
+</url>
+<url>
+  <loc>https://test.tisamed-oradea.ro/index.html</loc>
+  <lastmod>2023-08-01T21:35:23+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://test.tisamed-oradea.ro/despre/</loc>
```
Comment (georgebrata) [inline sitemap.xml:21]: Please use the live production domain (without test), like this: `https://tisamed-oradea.ro/`

File: sitemap.xml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  <lastmod>2023-08-01T21:35:21+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://test.tisamed-oradea.ro/servicii/</loc>
```
Comment (georgebrata) [inline sitemap.xml:26]: Same as above. Please update all lines below similarly.

**Comment by adonisjoe** [comment]
You can now merge.

**Review by georgebrata — APPROVED** [review]

**Pull request merged**